packages/analytics/amplify_analytics_pinpoint/example/tool/provision_integration_test_resources.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+pushd backend
+npm ci
+npm run setup
+npm run deploy
+popd
+
+BUCKET=$(jq -r .AnalyticsIntegrationTestStack.BucketName backend/outputs.json)
+aws --profile=${AWS_PROFILE:=default} s3 cp lib/amplifyconfiguration.dart s3://$BUCKET/amplifyconfiguration.dart
```

Comment:
nice, love the changes to this script. Also seemed to work well when I provided non-default AWS_PROFILE.